ros_buildfarm/templates/snippet/install_dh-python.Dockerfile.em

```diff
@@ -0,0 +1,4 @@
+@[if os_name == 'ubuntu' and os_code_name not in ['trusty', 'xenial', 'artful']]@
+@# Ubuntu Bionic doesn't ship dh-python with python3 anymore
+RUN for i in 1 2 3; do apt-get update && apt-get install -q -y dh-python && apt-get clean && break || if [[ $i < 3 ]]; then sleep 5; else false; fi; done
+@[end if]@
```

Comment:
In all the used cases the apt wrapper script seems to be available. There it should be used here instead of the handcrafted loop.